buildSrc/build.gradle.kts

```diff
@@ -36,11 +36,11 @@ dependencies {
     implementation(care.data4life.gradle.fhir.dependency.GradlePlugin.android)
 
     // dependency-updates.gradle.kts
-    implementation("com.github.ben-manes:gradle-versions-plugin:0.39.0")
+    implementation("com.github.ben-manes:gradle-versions-plugin:0.42.0")
     // download-scripts.gradle.kts
-    implementation("de.undercouch:gradle-download-task:4.1.2")
+    implementation("de.undercouch:gradle-download-task:5.1.1")
     // publishing.gradle.kts
-    implementation("org.eclipse.jgit:org.eclipse.jgit:5.11.0.202103091610-r")
+    implementation("org.eclipse.jgit:org.eclipse.jgit:6.2.0.202206071550-r")
     // versioning.gradle.kts
-    implementation("care.data4life.gradle.gitversion:gradle-git-version:0.12.4-d4l")
+    implementation("com.palantir.gradle.gitversion:gradle-git-version:0.15.0")
```

Comment:
Is this version working with nested git projects and submodules? That was the reason to have a forked version.

Reply:
As far as I am aware of it does - I used it for my personal project and did not run into any issues this far.